advisories/hackage/keter/HSEC-2024-0001.md

```diff
@@ -0,0 +1,30 @@
+```toml
+[advisory]
+id = "HSEC-2024-0001"
+cwe = [79]
+keywords = ["http", "xss", "rxss"]
```

Comment:
Please add `"historical"` to the keywords.

Reply:
done